src/components/Description/Description.js

```diff
@@ -2,16 +2,7 @@ import React from "react";
 import "./Description.css";
 
 const Description = (selectedMovie) => {
-  const {
-    overview,
-    releaseDate,
-    averageRating,
-    genre,
-    budget,
-    revenue,
-    runtime,
-    tagline,
-  } = selectedMovie;
+  const { overview, releaseDate, averageRating, genre, budget, revenue, runtime, tagline } = selectedMovie;
```

Comment:
Per our in-person conversation, we may revert to the standard convention of wrapping these restructured variables down.  A new chore will be added to the project board.